ovn_cluster.sh

```diff
@@ -649,6 +650,15 @@ EOF
     ip netns exec ovnfake-ext ip addr add 3000::b/64 dev ovnfake-ext
     ip netns exec ovnfake-ext ip link set ovnfake-ext up
     ip netns exec ovnfake-ext ip route add default via 172.16.0.1
+
+    echo "Creating a fake VM in the ovs bridge ${OVN_BR}"
+    ip netns add ovnfake-int
+    ovs-vsctl add-port ${OVN_BR} ovnfake-int -- set interface ovnfake-int type=internal
+    ip link set ovnfake-int netns ovnfake-int
+    ip netns exec ovnfake-int ip link set lo up
+    ip netns exec ovnfake-int ip link set ovnfake-int address 30:54:00:00:00:60
+    ip netns exec ovnfake-int ip addr add 170.168.0.1/${IP_CIDR} dev ovnfake-int
```

Comment:
how using something other than .1 here (maybe 170.168.0.50, if that is not taken)?
Maybe make it a variable like {OVNFAKE_INT_IP}?
Also, not sure if we would like to add a "route add default" and ipv6?
wdyt @numansiddique ?

Reply:
Thanks for the quick feedback @flavio-fernandes.
I'm not very familiar with this tool and given the CIDR length is dynamic, I was not sure if  choosing .50 was safe.
Since I saw the first assigned address is .2 and I did not see .1 being used anywhere, that's why I chose it.
But you guys are the experts here :)
WRT the route and ipv6: I did not see ipv6 addresses being added to chassis interfaces but if you think it's needed I'll add them